.gitignore

```diff
@@ -134,3 +134,12 @@ dmypy.json
 
 #sphinx docs
 _build
+
+#MC
+.vscode/settings.json
+.vscode/
+.project
+.pydevproject
+.vscode/launch.json
```

Comment:
```suggestion
```

You can remove this and `.vscode/settings.json` from this file because `.vscode/` ignores the whole folder

Reply:
Yes you're right. 
Done